MvvmCross-Forms/MvvmCross.Forms/Bindings/MvxBindExtension.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Text;
+using MvvmCross.Binding;
+using MvvmCross.Platform;
+using MvvmCross.Platform.Platform;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace MvvmCross.Forms.Bindings
+{
+    [ContentProperty("Path")]
+    public class MvxBindExtension : IMarkupExtension
+    {
+        public MvxBindExtension()
+        {
+            Mode = MvxBindingMode.Default;
+            Path = ".";
+        }
+
+        public string Path { get; set; }
+
+        public MvxBindingMode Mode { get; set; }
+
+        public string Converter { get; set; }
+
+        public string ConverterParameter { get; set; }
+
+        public string StringFormat { get; set; }
+
+        public string FallbackValue { get; set; }
+
+        public string CommandParameter { get; set; }
+
+        public object ProvideValue(IServiceProvider serviceProvider)
+        {
+            IProvideValueTarget target = serviceProvider.GetService(typeof(IProvideValueTarget)) as IProvideValueTarget;
```

Comment:
This looks strange to me. You are requesting a `IProvideValueTarget` then casting it to `IProvideValueTarget` right after. Could probably be done a lot cleaner.